tests/user_commands/management/commands/dance.py

```diff
@@ -5,6 +5,7 @@ class Command(BaseCommand):
     help = "Dance around like a madman."
     args = ''
     requires_system_checks = True
+    private_options = ('stdout', 'opt_3', 'skip_checks')
```

Comment:
Oh, I see. Well it shouldn't be added to `private_options`. The validation should be adjusted to account for 2cc8ffe258008096a70791115b2daa12f0ef0192.